code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -93,27 +93,29 @@ There are several things that need to be remembered:
 #define SHOES_LAYER       7
 #define GLOVES_LAYER      8
 #define BELT_LAYER        9
-#define SUIT_LAYER       10
-#define TAIL_LAYER       11
-#define GLASSES_LAYER    12
-#define BELT_LAYER_ALT   13
-#define SUIT_STORE_LAYER 14
-#define BACK_LAYER       15
-#define HAIR_LAYER       16
-#define EARS_LAYER       17
-#define FACEMASK_LAYER   18
-#define HEAD_LAYER       19
-#define COLLAR_LAYER     20
-#define HANDCUFF_LAYER   21
-#define LEGCUFF_LAYER    22
-#define L_HAND_LAYER     23
-#define R_HAND_LAYER     24
-#define FIRE_LAYER       25		//If you're on fire
-#define TOTAL_LAYERS     25
+#define TAIL_SOUTH_LAYER 10
+#define SUIT_LAYER       11
+#define TAIL_NORTH_LAYER 12
+#define GLASSES_LAYER    13
+#define BELT_LAYER_ALT   14
+#define SUIT_STORE_LAYER 15
+#define BACK_LAYER       16
+#define HAIR_LAYER       17
+#define EARS_LAYER       18
+#define FACEMASK_LAYER   19
+#define HEAD_LAYER       20
+#define COLLAR_LAYER     21
+#define HANDCUFF_LAYER   22
+#define LEGCUFF_LAYER    23
+#define L_HAND_LAYER     24
+#define R_HAND_LAYER     25
+#define FIRE_LAYER       26		//If you're on fire
+#define TOTAL_LAYERS     26
 //////////////////////////////////
 
 #define UNDERSCORE_OR_NULL(target) "[target ? "[target]_" : ""]"
 #define GET_BODY_TYPE (cached_bodytype || (cached_bodytype = species.get_bodytype()))
+#define GET_TAIL_LAYER (dir == NORTH ? TAIL_NORTH_LAYER : TAIL_SOUTH_LAYER)

```

Comment:
I don't see this being undef'd anywhere.

Reply:
fixed